OPERATOR.rst

```diff
@@ -260,14 +262,15 @@ To do a promotion:
    permanently solve the issues, revert the hotfix on ``prod``. Check the
    contributing guide for specifics on procedure.
 
-#. ::
+#. Then run::
 
       git checkout develop
```

Comment:
… this is redundant. I think we should move the `checkout` and `pull` parts to above step. The post-condition of that step should be that prod and develop are up-to-date and that `develop` is checked out.

Reply:
From PL: moved the `checkout` `pull` commands, added `develop` as the upstream for new branch.